sphinx/themes/basic/static/doctools.js

```diff
@@ -294,20 +294,26 @@ var Documentation = {
       if (activeElementType !== 'TEXTAREA' && activeElementType !== 'INPUT' && activeElementType !== 'SELECT'
           && activeElementType !== 'BUTTON' && !event.altKey && !event.ctrlKey && !event.metaKey
           && !event.shiftKey) {
-        switch (event.keyCode) {
-          case 37: // left
+        switch (event.key) {
+          case 'ArrowLeft':
             var prevHref = $('link[rel="prev"]').prop('href');
             if (prevHref) {
               window.location.href = prevHref;
               return false;
             }
-            break;
-          case 39: // right
+          case 'ArrowRight':
             var nextHref = $('link[rel="next"]').prop('href');
             if (nextHref) {
               window.location.href = nextHref;
               return false;
             }
+          case '/':
+          case 's':
+            $('input[name=q]').focus();
+            return false;
+          case 'c':
+            newurl = window.location.href.replace(new RegExp('\\?highlight=[^#]*'), '');
+            window.location.href = newurl;
```

Comment:
There is already existing code for this functionality:

https://github.com/sphinx-doc/sphinx/blob/c66b3ad519ed91e79205ee973bfdba17fc06453f/sphinx/themes/basic/static/doctools.js#L261-L267

What you are suggesting is causing a page reload, which I think isn't desirable.

Reply:
You are right, I'm not using the existing JS functionality. Good point!